src/bp/ui-admin/src/Pages/Workspace/Bots/index.tsx

```diff
@@ -171,6 +178,22 @@ class Bots extends Component<Props> {
     toastSuccess('Rollback success')
   }
 
+  handleEditStageSuccess = () => {
+    this.props.fetchBots()
+    toastSuccess('Stage saved successfully')
```

Comment:
why not display this success toast within the modal component, like the failure toast ?